janusgraph-solr/pom.xml

```diff
@@ -56,19 +54,27 @@
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-solrj</artifactId>
             <version>${solr.version}</version>
+            <exclusions>
+                <!-- Use jcl-over-slf4j from gremlin-core -->
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>jcl-over-slf4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.solr</groupId>
```

Comment:
@wojciechwojcik Understood. I'll take #310 off this issue so it can continue to track separately the removal of solr-core from janusgraph-solr. Once this PR is merged please update your issue title and description to remove references to the embedded Lucene error (assuming it's resolved).